test/integration/content-explorer/ContentExplorer.int.test.js

```diff
@@ -276,5 +292,41 @@ describe('ContentExplorer', () => {
             helpers.getClosePreviewButton().click();
             cy.getByAriaLabel(localize('be.preview')).should('not.exist');
         });
+
+        it('Should increase number of columns', () => {
+            helpers.getGridViewColumnCount().then(numColumns => {
+                helpers.getSliderMinusButton().click();
+                helpers.getGridViewColumnCount().should('equal', numColumns + 1);
+            });
+        });
+
+        it('Should decrease number of columns', () => {
+            helpers.getGridViewColumnCount().then(numColumns => {
```

Comment:
We should favor larger, more complex cases for our E2E tests, due to the overhead in setting up and tearing down each case. Should these be unit tests, instead?